gdal/doc/source/user/ogr_sql_dialect.rst

```diff
@@ -535,12 +536,29 @@ the feature id, but it may be explicitly included using a syntax like:
 
     SELECT FID, * FROM nation
 
+Geometry field
+++++++++++++++
+
+The OGR SQL dialect adds the geometry field of the datasource to the result set
+by default. Users do not need to select the geometry explicitly but it is still
+possible to do so. Common use case is when geometry is the only field that is needed.
+In this case the name of the geometry field to be used in the SQL statement is the
+name returned by :cpp:func:`OGRLayer::getGeometryColumn()`. If the method returns
+an empty string then a special name "_ogr_geometry_" must be used. The name begins
+with an underscore and SQL syntax requires that it must appear between double quotes.
+In addition the command line interpreter may require that double quotes are escaped
+and the final SELECT statement could look like:
+
+.. code-block::
+
+    SELECT \"_ogr_geometry_\" FROM nation
```

Comment:
```/__w/1/s/gdal/doc/source/user/ogr_sql_dialect.rst:552: WARNING: Could not lex literal_block as "sql". Highlighting skipped```
The escaping isn't required by the SQL syntax highlighing. This is a shell specific issue
```suggestion
    SELECT "_ogr_geometry_" FROM nation
```